app/internal/window/os_js.go

```diff
@@ -417,22 +418,17 @@ func (w *window) funcOf(f func(this js.Value, args []js.Value) interface{}) js.F
 func (w *window) animCallback() {
 	w.mu.Lock()
 	anim := w.animating
-	if anim {
-		w.requestAnimationFrame.Invoke(w.redraw)
-	}
 	w.mu.Unlock()
 	if anim {
 		w.draw(false)
 	}
+	w.requestAnimationFrame.Invoke(w.redraw)
 }
 
 func (w *window) SetAnimating(anim bool) {
 	w.mu.Lock()
-	defer w.mu.Unlock()
-	if anim && !w.animating {
-		w.requestAnimationFrame.Invoke(w.redraw)
```

Comment:
Without requestAnimationFrame here, how will animation start? With your change, SetAnimating only sets w.animating, and that's it.

Reply:
It starts inside NewWindow, after the w.draw(true), 41d9101#diff-abf70745e7c00ff72bc8babcc591453b6cde31a320dbb2bc50c13872115855faR85.